### DISCUSSION TITLE
Repeating LSI after project subsetting

### DISCUSSION THREAD

**Discussion by sylestiel** [GreenleafLab/ArchR#623] [Questions / Documentation] [answered]
If I wish to analyze a subset of clusters from the original dataset do I need to repeat the LSI step?

Also do i need to filter cells and enable doublet removal or is that done just once?

Thanks,

**Comment by rcorces** [comment, marked as answer]
1) Doublet removal is done on a per-arrow basis at the start of a project.

2) The workflow after project subsetting depends on what you want to accomplish. If you are subsetting to gain better resolution of specific clusters and you are interested in further subdividing those clusters, then it makes the most sense to me to repeat dimensionality reduction on the subset and then repeat cluster identification etc.

**Reply by sylestiel** [reply]
Thank you!
